llvm/test/Analysis/CostModel/SystemZ/load-to-trunc.ll

```diff
@@ -1,27 +0,0 @@
-; NOTE: Assertions have been autogenerated by utils/update_analyze_test_checks.py
```

Comment:
Since this is build for z10, shouldn't all those costs remain unchanged?  Why are you deleting this test?

Reply:
oops - sorry, I didn't realize that was a z10 test and thought I had replaced it with new tests in i128-cmp-ext-conv.ll. Adding it back.